kernel/src/protocols/core.rs

```diff
@@ -86,6 +86,11 @@ fn core_create_vcpu(params: &RequestParams) -> Result<(), SvsmReqError> {
         return Err(SvsmReqError::invalid_address());
     }
 
+    // Check whether VMSA page and CAA region overlap
+    if overlap(paddr, paddr + PAGE_SIZE - 1, pcaa, pcaa + 7) {
```

Comment:
Maybe this is clearer:
```suggestion
    if overlap(paddr, paddr + PAGE_SIZE - 1, pcaa, pcaa + size_of::<SvsmCaa> - 1) {
```

Reply:
Makes sense, changed that.